src/lib/homeassistant-weather-icons/index.ts

```diff
@@ -0,0 +1,188 @@
+import { SVGTemplateResult, svg, css } from "lit";
+
+const cloudyStates = new Set<string>([
```

Comment:
I copied this as-is. Do we really need this many states? Where can I read more about this topic?